cmd/microservice-fanfare-fluidity-money/util.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+package main
+
+import (
+	"math"
+	"math/big"
+	"strings"
+)
+
+func pow10(x int) *big.Rat {
+	return new(big.Rat).SetFloat64(math.Pow10(x))
+}
+
+func cleanAddress(x string) string {
```

Comment:
Function name doesn't align with what it does. Seems generally redundant - why not just call `strings.ToLower` directly?

Reply:
Having a function that wraps the lowering of the string adds extra functional context to the act of doing so, as well as makes the idea of "cleaning" more abstract.